src/services/item/types.ts

```diff
@@ -23,4 +23,5 @@ export type ItemSearchParams = {
   sortBy?: SortBy;
   ordering?: Ordering;
   permissions?: PermissionLevel[];
+  itemType?: (typeof ItemType)[keyof typeof ItemType];
```

Comment:
use `UnionOfConst<typeof ItemType` ?

Reply:
Yes I  updated the code, now I use unionOfConst and we can use an array.